podpac/core/coordinates/utils.py

```diff
@@ -487,10 +487,7 @@ def lower_precision_time_bounds(my_bounds, other_bounds, outer):
     if not isinstance(my_bounds[0], np.datetime64) or not isinstance(my_bounds[1], np.datetime64):
         raise TypeError("Native bounds should be of type np.datetime64 when selecting data using:", str(other_bounds))
 
-    mine = np.timedelta64(1, my_bounds[0].dtype.name.replace("datetime64", "")[1:-1])
-    your = np.timedelta64(1, other_bounds[0].dtype.name.replace("datetime64", "")[1:-1])
-
-    if mine > your and outer:
+    if my_bounds[0].dtype < other_bounds[0].dtype and outer:
```

Comment:
that's a lot cleaner... i suspect the previous code had my dirty fingerprints on them. :)